backend/FileSystem/FileManager.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\FileSystem;
+
+use Exception;
+
+class FileManager
+{
+    public function __construct(
+        protected DirectoryManager $directoryManager
+    ) {
+    }
+
+    public function create(string $relativePath, string $filename = null, mixed $content = ""): void
```

Comment:
Why `$filename` can be null?

Reply:
I intended this function to create both files or folders, even without files.

Maybe it could be used to create directory structure and then detect if any folder is missing some data.

Reply:
Maybe add flag/param `$withoutFile = false` will be better?